tests/settings.py

```diff
@@ -94,6 +94,11 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+# middleware list for newer djangos
+MIDDLEWARE = list(MIDDLEWARE_CLASSES)
+MIDDLEWARE.remove('django.contrib.auth.middleware.SessionAuthenticationMiddleware')
```

Comment:
Why this operation is needed?

Reply:
It seems that session auth middleware is a no-op since django 1.10. In recent versions it is not defined anymore.

https://docs.djangoproject.com/en/3.0/releases/2.0/#miscellaneous

will add a comment in the code to make that clear + maybe a switch with explicite django version check